schema/pipeline.yaml

```diff
@@ -25,6 +25,13 @@ properties:
     additionalProperties: 
       $ref: ./input.yaml
 
+  repository:
+    type: object
+    description: Repository defines the pipeline repo parameters.
+    additionalProperties: 
+      $ref: ./repository.yaml
```

Comment:
This is producing a `map[string]*Reference`. Should this be the following, instead?

```
  repository:
    description: Repository defines the pipeline repo parameters.
    $ref: ./repository.yaml
```